README.md

```diff
@@ -35,10 +43,8 @@ Programming languages where HTML defines the content and structure of a webpage,
 NFC (Near Field Communication) chips are small integrated circuits that enable short-range wireless communication between devices, facilitating contactless data transfer and transactions.
 
 ## Project Setup/Installation
-### Create a Replit Account.
-[Creating a Replit Account.pdf](https://github.com/TAP-GGC/TapIn/blob/main/documents/Creating%20a%20Replit%20Account.pdf)
-### Create a Website on Replit
-[Creating a Website on Replit.pdf](https://github.com/TAP-GGC/TapIn/blob/main/documents/Creating%20a%20Website%20on%20Replit.pdf)
+### Create a Website on Glitch
+[Creating a Website on Glitch.pdf](https://github.com/aeristran/TapIn/blob/main/documents/Create%20your%20Website%20on%20Glitch.pdf)
```

Comment:
use relative links here so it points to the folders regardless whose account it's in. just change it to start from `documents/...`